src/routes/blog.js

```diff
@@ -1,24 +1,51 @@
 const express = require("express");
-const { MongoClient } = require("mongodb");
+const bodyParser = require("body-parser");
```

Comment:
Why should i add body-parser to index.js then im not using it in there?

Reply:
Just like in the course.
Something, which makes sense to have for the whole backend.
We might use it afterwards, and it will already be there.
Otherwise, there will be a duplication :D